tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -38,6 +39,68 @@ class BackendCorrectnessTest : public ::testing::TestWithParam<BackendKind> {
 
 class CPUOnly : public BackendCorrectnessTest {};
 
+namespace {
+
+/// Clone, profile, and run \p origF given the \p ctx and \p EE. \returns the
+/// quantization parameters from the profile.
+static std::vector<NodeQuantizationInfo>
+profileAndGetNodeQuantizationInfo(Context &ctx, ExecutionEngine &EE,
+                                  Function *origF) {
+  Function *profileF = origF->clone("profileF");
```

Comment:
you do not really need to clone this F as it'll be taken care of as part of the ```profileQuantization```

Reply:
Thanks, done!